Dockerfile

```diff
@@ -36,6 +36,16 @@ RUN curl -L -o ~/bin/solc-v0.5.1 https://github.com/ethereum/solidity/releases/d
 RUN curl -L -o ~/bin/solc-v0.5.7 https://github.com/ethereum/solidity/releases/download/v0.5.7/solc-static-linux && chmod +x ~/bin/solc-v0.5.7
 RUN curl -L -o ~/bin/solc-v0.5.8 https://github.com/ethereum/solidity/releases/download/v0.5.8/solc-static-linux && chmod +x ~/bin/solc-v0.5.8
 
+# Donload unzip
+RUN sudo apt-get install -y unzip
+
+# Download openethereum 3.0.1.
+RUN curl -L -o openethereum_zip https://github.com/openethereum/openethereum/releases/download/v3.0.1/openethereum-linux-v3.0.1.zip
```

Comment:
why do you not use ADD here?

Reply:
If I use `ADD` I get `openethereum_zip: Permission denied` on the next line trying to calculate the sha sum, which I do not understand to be honest. Plus, right above the installation of solc uses `RUN curl -L -o`.